test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -2074,6 +2080,14 @@ protected boolean addMockTransportService() {
         return true;
     }
 
+    /**
+     * Iff this returns true concurrent search is favour. Otherwise runs with default set up.
+     * The default is randomized.
+     */
+    protected boolean addConcurrentSearch() {
+        return randomBoolean();
```

Comment:
do you have a preview of cases where concurrency will need to be disabled?

Reply:
not at the moment.